### DISCUSSION TITLE
Kimai stops recording activity when another activity is loged in weekly working hours

### DISCUSSION THREAD

**Discussion by martin-chalupa** [kimai/kimai#4988] [General] [answered]
Hello, I would like the employees to track attendence using play button and all other activities by manually loging them in weekly working hours. But when attendence recording is running, I cant log any other entries cause they will cause the running activity to stop. Is this suppose to happen? Is there a way around this? Thanks

**Comment by kevinpapst** [comment, marked as answer]
Without looking at the code: if the running entry is within the edited week then yes.
This view assumes every entry has a duration. A duration means there is start and stop.
